components/no_results_indicator/no_results_indicator.tsx

```diff
@@ -66,7 +76,10 @@ const NoResultsIndicator = (props: Props) => {
                 />
             </div>
             <FormattedMessage
```

Comment:
Is this one of those react-intl magic things? where `FormattedMessage` is able to handle HTML Messages? Should this be `FormattedHTMLMessage` instead? orr 🤔

Reply:
No for some reason FormattedHTMLMessage doesn't support icon 🙄 let alone custom elements so it had to be done this way.

Reply:
@nevyangelova oh okay so `FormattedHTMLMessage` is not being used then?

Reply:
FormattedHTMLMessage is going away in 4.x, anyways. And FormattedMessage will be able to accept [rich-text formatting](https://formatjs.io/docs/react-intl/components#rich-text-formatting) in 4.x! (Working on an upgrade PR [over here](https://github.com/mattermost/mattermost-webapp/pull/5428).)

Reply:
@ali-farooq0 I know what you meant. I updated the variable name to 'subtitleValues' since it was deceiving.